### DISCUSSION TITLE
Hello new here. Looking for some python help

### DISCUSSION THREAD

**Discussion by Burnfire23** [codewars/codewars.com#3013] [General]
New to Python was hoping to team up with some people to learn a little. I have some knowledge but not much. 

Burnfire420#9462 - Discord

**Comment by max-codex** [comment]
yes tell me

**Comment by hobovsky** [comment]
To get in touch with other users, please join Codewars Discord server. You can find something interesting there, on `#beginners` or `#python` channels.
